task_executor/scripts/scheduled_task_executor.py

```diff
@@ -259,11 +262,12 @@ def call_scheduler(self, tasks, earliest_start, current_id=0):
         Calls scheduler. Reorders the list of tasks in execution order with their execution times set. 
         
         """
-        resp = self.schedule_srv(tasks, earliest_start, current_id, self.get_duration_matrix(tasks))
+        resp = self.schedule_srv(tasks, earliest_start, current_id, self.get_duration_matrix(tasks)) # Lenka note: why does the service need earliest start and current id?
```

Comment:
Yep, but that informations if propagated by earliest_start. I think the current_id is doing nothing right now